src/LA/system_solvers_large.cpp

```diff
@@ -2488,6 +2520,29 @@ void SystemSolver::createVector(int blockSize, Vec *vector) const
     }
 }
 
+/*!
+ * Create a nest vector.
+ *
+ * \param blockSize is the block size of the vector
+ * \param nNestedItem is the number of nested item blocks
+ * \param subVectors are the subvectors, empty subvectors can be passed using nullptr
```

Comment:
Could we make the arguments name more clear? Is nNestedItem the number of subVectors?

Reply:
And for matrix too?

Reply:
This is the updated documentation:
```
/*!
 * Create a nest vector.
 *
 * \param blockSize is the block size of the vector
 * \param nestSize is the number of subvectors that will be contained in the nest
 * \param subVectors are the subvectors, empty subvectors can be passed using nullptr
 * \param vector on output will contain the newly created vector
 */
```

Reply:
```
/*!
 * Create a nest matrix.
 *
 * \param rowBlockSize is the row block size of the matrix
 * \param colBlockSize is the column block size of the matrix
 * \param nNestRows is the number of rows in the nest, i.e. the number of submatrices along the
 * rows of the nest
 * \param nNestCols is the number of columns in the nest, i.e. the number of submatrices along
 * the columns of the nest
 * \param subMatrices are the submatrices stored in row-major order, empty submatrices can
 * be passed using nullptr
 * \param vector on output will contain the newly created matrix
 */
```